pkg/provisioner/templates/kubernetes.go

```diff
@@ -147,12 +156,54 @@ echo "you can now access the cluster with:"
 echo "ssh -i <your-private-key> ubuntu@${INSTANCE_ENDPOINT_HOST}"
 `
 
+const NVKindTemplate = `
+
+# Go
+# Set GO version
+GO_VERSION="1.23.6"
+wget https://go.dev/dl/go${GO_VERSION}.linux-amd64.tar.gz -O /tmp/go${GO_VERSION}.linux-amd64.tar.gz
+sudo rm -rf /usr/local/go
+sudo tar -C /usr/local -xzf /tmp/go${GO_VERSION}.linux-amd64.tar.gz
```

Comment:
This really does seem like a strange place to install go.

Reply:
since there's no official release for `nvkind` we are installing it via `go install` so this step is only needed for `nvkind` but not for `kind`